doc/yarn/dashboard.py

```diff
@@ -0,0 +1,22 @@
+import skein
```

Comment:
this seems to be the same file from `doc_code/yard/dashboard.py` -- is that intentional?

Reply:
Yes. I noticed there are same files under `doc/cluster/doc_code/yarn/` and `doc/yarn`, such as `example.py` and `ray-skein.yaml`, so I add same changes under two folder.